.agents/skills/consolidation/nemoclaw-sprint-review/collect_sprint_items.py

```
@@ -0,0 +1,132 @@
+                field_name = fv.get('field', {}).get('name', '')
+                if field_name == 'Sprint' and fv.get('title') == target_sprint:
+                    sprint = target_sprint
+                if field_name == 'Status':
+                    status = fv.get('name')
+
+            if sprint == target_sprint:
+                num = content.get('number', '?')
+                title = content.get('title', '?')
+                state = content.get('state', '?')
+                assignees = [a['login'] for a in content.get('assignees', {}).get('nodes', [])]
+                labels = [l['name'] for l in content.get('labels', {}).get('nodes', [])]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)